addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -747,15 +748,20 @@ function factory(dependencies) {
                     const channelNameWithIcon = channelIcon + channelName;
                     notificationTitle = _.str.sprintf(
                         this.env._t("%s from %s"),
-                        owl.utils.escape(authorName),
+                        escapedAuthorName,
                         channelNameWithIcon
                     );
                 } else {
-                    notificationTitle = owl.utils.escape(authorName);
+                    notificationTitle = escapedAuthorName;
                 }
             }
-            const notificationContent = message.prettyBody.substr(0, PREVIEW_MSG_MAX_SIZE);
-            this.env.services['bus_service'].sendNotification(notificationTitle, notificationContent);
```

Comment:
Here we must keep using the notification from `bus_service`, it allows to select between native notification or just inside Odoo. `notification` service itself is always just inside Odoo. -> New message is important enough to deserve a native notification if possible.

What I mean by previous suggestion is to update the API of `sendNotification` from `bus_service` (accepting `options` parameter, same API as `displayNotification`) to allow the extra param that are necessary here.